scripts/test-renode.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
+
+SCRIPT_SRC=$(realpath ${BASH_SOURCE[0]})
+SCRIPT_DIR=$(dirname $SCRIPT_SRC)
+TOP_DIR=$(realpath $SCRIPT_DIR/..)
+if [[ "${BASH_SOURCE[0]}" != "${0}" ]]; then
+	echo "You must run this script, rather then try to source it."
+	echo "$SCRIPT_SRC"
+	exit 1
+fi
+
+source $SCRIPT_DIR/build-common.sh
+
+init
+
+case $CPU in
+	vexriscv)
+    		CPU_TYPE="VexRiscv"
+		;;
+	picorv32)
+    		CPU_TYPE="PicoRV32"
+		;;
+	*)
+		echo "Simulating the current configuration is not supported in Renode yet - skipping the test"
+		exit 0
+		;;
+esac
+
+if [ "$PLATFORM" == "mimas_a7" ] || [ "$PLATFORM" == "netv2" ] || [ "$PLATFORM" == "pano_logic_g2" ]; then
+    # the test load binaries into the flash to avoid using netboot on CI server
+    echo "$PLATFORM platform does not have flash memory - skipping the test"
+    exit 0
+fi
+
+if [ "$FIRMWARE" == "zephyr" ]; then
+    if [ "$PLATFORM" == "icebreaker" ]; then
+        # running Zephyr firmware directly from flash is not supported at the moment
+        # as it requires to enable XIP and include flash section in the DTS;
+        # TODO: add this in the future
+        echo "Running $FIRMWARE directly from flash is currently not supported - skipping the test"
```

Comment:
Some platforms boot directly from flash (XIP), what is not currently supported by the Zephyr binaries built by `litex-buildenv`.

Once LiteX spi flash dirver is merged to Zephyr, `scripts/build-zephyr.sh` script can be extended to allow XIP and the limitation in this file can be removed.

Reply:
Can you log a tracking but about this?

Reply:
https://github.com/timvideos/litex-buildenv/issues/349